sig-security/security-audit-2021/RFP.md

```diff
@@ -127,3 +127,12 @@ The audit should result in the following deliverables, which will be made public
 ### Do we need to use our own hardware and infrastructure or should we use a cloud?
 
 Strong preference would be for the vendor to provide their own infrastructure or use a public cloud provider, just NOT a managed offering like GKE or EKS. The reasoning is to prevent accidentally auditing a cloud provider's kubernetes service instead of kubernetes/kubernetes. Depending on the scope and approach, it may make sense to use a local cluster (e.g. kind) for API fuzzing and anything that doesn't impact the underlying OS, and is an easy to use repeatable setup (see Methodology above).
+
+### What is the intellectual property ownership of the report and all work product?
+
+The report must be licensed under CC BY 4.0 based on the charter. (https://github.com/cncf/foundation/blob/master/charter.md#11-ip-policy - see (f))
+Separately, any code used needs to be under Apache 2.0, see https://github.com/cncf/foundation/blob/master/charter.md#11-ip-policy - sections (e) and (d). 
```

Comment:
```suggestion
Separately, any code used needs to be under the Apache License, version 2.0. Please refer to [sections 11.(e) and (d) in the CNCF Charter](https://github.com/cncf/foundation/blob/master/charter.md#11-ip-policy).
```

Reply:
Suggested change to align language in the CNCF charter and add a markdown hyperlink